tikv-configuration-file.md

```diff
@@ -1110,9 +1106,9 @@ Configuration items related to `rocksdb.defaultcf`, `rocksdb.writecf`, and `rock
 ### `compaction-pri`
 
 + The priority type of compaction
-+ Optional values: `0` (`ByCompensatedSize`), `1` (`OldestLargestSeqFirst`), `2` (`OldestSmallestSeqFirst`), `3` (`MinOverlappingRatio`)
-+ Default value for `defaultcf` and `writecf`: `3`
-+ Default value for `lockcf`: `0`
++ Optional values: `"by-compensated-size"`, `"oldest-largest-seq-first"`, `"oldest-smallest-seq-first"`, `"min-overlapping-ratio"`
```

Comment:
to be explained

Reply:
```suggestion
+ Optional values: 
    - `"by-compensated-size"`: prioritize the compaction on large files in order of file size
    - `"oldest-largest-seq-first"`: perform the compaction on files with data whose latest update time is old in chronological order. **Only** set this value when updating hot keys in small ranges.
    - `"oldest-smallest-seq-first"`: prioritize compaction on files with ranges that are not compacted to the next level for a long time in chronological order. If you randomly update hot keys across the key space, setting this value can slightly reduce write amplification.
    - `"min-overlapping-ratio"`: prioritize compaction on files with a high overlap ratio. When the file size in different levels is small (which means the value of  `the file size in the next level` ÷ `the file size in this level` is small), TiKV compacts the file first. In many cases, setting this value can effectively reduce write amplification.
```

Reply:
@en-jin19 What is the input of this content?